src/main/kotlin/data/entity/Guild.kt

```diff
@@ -0,0 +1,40 @@
+package tech.trip_kun.sinon.data.entity
+
+import com.j256.ormlite.field.DatabaseField
+import com.j256.ormlite.field.ForeignCollectionField
+import com.j256.ormlite.table.DatabaseTable
+import tech.trip_kun.sinon.annotation.ReflectionNoArg
+
+@ReflectionNoArg
+@DatabaseTable(tableName = "guilds")
+class Guild(@DatabaseField(id = true) var id: Long) {
+    @DatabaseField(canBeNull = false)
+    var currencyEnabled: Boolean = false
+
+    @DatabaseField(canBeNull = false)
+    var starboardChannelId: Long = 0
```

Comment:
It's because discord snowflake IDs are never 0 that we can use it. Zero is our "not set" value, This PR isn't yet complete so that will become more clear as the code utilizing these database structures is built. All usages will check for if the starboardChannelId is zero and act accordinly.